tonytanaka/index.js

```diff
@@ -0,0 +1,37 @@
+// 1
+const mult = (x, y) => x * y;
+console.log("mult ==>", mult(2, 3));
+
+// 2
+oddNumbers = [];
+const double = (arr) => {
+  for (let x = 0; x < arr.length; x++) {
+    if (arr[x] % 2 != 0) {
+      oddNumbers.push(arr[x]);
```

Comment:
You have to multiply oddNumbers to 2 when you push them in oddNumbers

Reply:
oh I forgot that